extensions/extensions-docker/aissemble-pipeline-invocation/pom.xml

```diff
@@ -24,58 +24,9 @@
             <version>${project.version}</version>
         </dependency>
     </dependencies>
-    <profiles>
-        <profile>
-            <id>ci</id>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>io.fabric8</groupId>
-                        <artifactId>docker-maven-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <id>default-push</id>
-                                <configuration>
-                                    <images>
-                                        <image>
-                                            <build>
-                                                <args>
-                                                    <AISSEMBLE_HELM_REPO>${aissemble.helm.repo}</AISSEMBLE_HELM_REPO>
```

Comment:
I: Removed as this build argument is not used in this dockerfile

Reply:
Just curious, why was the helm repo url being given to docker in the first place? I know you said it was unused, but I wonder why this was simply just a mistake from before?

Reply:
Seems like it was mistakenly added in the initial build out of the invocation service and was never caught.